crates/storage/src/tables.rs

```diff
@@ -54,7 +54,7 @@ impl Mappable for ContractsLatestUtxo {
     type OwnedKey = ContractId;
     /// The latest UTXO id.
     type Value = Self::OwnedValue;
-    type OwnedValue = UtxoId;
+    type OwnedValue = (UtxoId, fuel_core_types::fuel_tx::TxPointer);
```

Comment:
Let's replace it with a new type like `ContractsBalance` or `ContractsCoin` or `ContractsState`

Reply:
done